### ISSUE TITLE
a11y warnings

### ISSUE THREAD

**Issue by Schmell** [huntabyte/shadcn-svelte#1540]
### Describe the bug

I get accesabilities warnings when using shadcn/bit-ui components.

```[vite-plugin-svelte] src/lib/components/ui/button/button.svelte:63:20  `a11y-missing-attribute` is no longer valid — please use `a11y_missing_attribute` instead```

After some disscussion [#1512](https://github.com/huntabyte/shadcn-svelte/discussions/1512) It has been determined that this happens when you add paraglide with `npx sv`

### Reproduction

i can't make `npx sv` work with stackblitz and if i try to  manually install  paraglide it does not seem to play well with stack blitz.
So i applogize, but all i did was use `npx sv create` and added paraglide

### Logs

```bash
10:41:39 a.m. [vite-plugin-svelte] src/lib/components/ui/button/button.svelte:62:20 `a11y-missing-attribute` is no longer valid — please use `a11y_missing_attribute` instead   
10:41:39 a.m. [vite-plugin-svelte] src/lib/components/ui/button/button.svelte:71:20 `a11y-missing-attribute` is no longer valid — please use `a11y_missing_attribute` instead   
undefined /
10:41:41 a.m. [vite-plugin-svelte] src/lib/components/nav-main.svelte:65:31 `a11y-missing-attribute` is no longer valid — please use `a11y_missing_attribute` instead
10:41:41 a.m. [vite-plugin-svelte] src/lib/components/nav-main.svelte:76:32 `a11y-missing-attribute` is no longer valid — please use `a11y_missing_attribute` instead
10:41:41 a.m. [vite-plugin-svelte] src/lib/components/nav-projects.svelte:37:25 `a11y-missing-attribute` is no longer valid — please use `a11y_missing_attribute` instead       
10:41:41 a.m. [vite-plugin-svelte] src/lib/components/ui/breadcrumb/breadcrumb-link.svelte:33:20 `a11y-missing-attribute` is no longer valid — please use `a11y_missing_attribute` instead
10:41:42 a.m. [vite-plugin-svelte] src/lib/components/ui/sidebar/sidebar-group-action.svelte:38:20 `a11y-missing-attribute` is no longer valid — please use `a11y_missing_attribute` instead
10:41:42 a.m. [vite-plugin-svelte] src/lib/components/ui/sidebar/sidebar-menu-action.svelte:45:20 `a11y-missing-attribute` is no longer valid — please use `a11y_missing_attribute` instead
10:41:42 a.m. [vite-plugin-svelte] src/lib/components/ui/sidebar/sidebar-menu-button.svelte:79:21 `a11y-missing-attribute` is no longer valid — please use `a11y_missing_attribute` instead
10:41:42 a.m. [vite-plugin-svelte] src/lib/components/ui/sidebar/sidebar-menu-sub-button.svelte:45:20 `a11y-missing-attribute` is no longer valid — please use `a11y_missing_attribute` instead
10:41:42 a.m. [vite-plugin-svelte] src/lib/components/ui/sidebar/sidebar-rail.svelte:22:19 `a11y-missing-attribute` is no longer valid — please use `a11y_missing_attribute` instead
10:41:42 a.m. [vite-plugin-svelte] src/lib/components/ui/button/button.svelte:62:20 `a11y-missing-attribute` is no longer valid — please use `a11y_missing_attribute` instead   
10:41:42 a.m. [vite-plugin-svelte] src/lib/components/ui/button/button.svelte:71:20 `a11y-missing-attribute` is no longer valid — please use `a11y_missing_attribute` instead   
10:41:48 a.m. [vite-plugin-svelte] src/lib/components/nav-main.svelte:65:31 `a11y-missing-attribute` is no longer valid — please use `a11y_missing_attribute` instead
10:41:48 a.m. [vite-plugin-svelte] src/lib/components/nav-main.svelte:76:32 `a11y-missing-attribute` is no longer valid — please use `a11y_missing_attribute` instead
10:41:48 a.m. [vite-plugin-svelte] src/lib/components/nav-projects.svelte:37:25 `a11y-missing-attribute` is no longer valid — please use `a11y_missing_attribute` instead       
10:41:48 a.m. [vite-plugin-svelte] src/lib/components/ui/breadcrumb/breadcrumb-link.svelte:33:20 `a11y-missing-attribute` is no longer valid — please use `a11y_missing_attribute` instead
10:41:48 a.m. [vite-plugin-svelte] src/lib/components/ui/sidebar/sidebar-group-action.svelte:38:20 `a11y-missing-attribute` is no longer valid — please use `a11y_missing_attribute` instead
10:41:48 a.m. [vite-plugin-svelte] src/lib/components/ui/sidebar/sidebar-menu-action.svelte:45:20 `a11y-missing-attribute` is no longer valid — please use `a11y_missing_attribute` instead
10:41:48 a.m. [vite-plugin-svelte] src/lib/components/ui/sidebar/sidebar-menu-button.svelte:79:21 `a11y-missing-attribute` is no longer valid — please use `a11y_missing_attribute` instead
10:41:48 a.m. [vite-plugin-svelte] src/lib/components/ui/sidebar/sidebar-menu-sub-button.svelte:45:20 `a11y-missing-attribute` is no longer valid — please use `a11y_missing_attribute` instead
10:41:49 a.m. [vite-plugin-svelte] src/lib/components/ui/sidebar/sidebar-rail.svelte:22:19 `a11y-missing-attribute` is no longer valid — please use `a11y_missing_attribute` instead
10:42:16 a.m. [vite-plugin-svelte] src/lib/components/ui/badge/badge.svelte:47:19 `a11y-missing-attribute` is no longer valid — please use `a11y_missing_attribute` instead
```

### System Info

```bash
System:
    OS: Windows 10 10.0.19045
    CPU: (4) x64 Intel(R) Core(TM) i5-7200U CPU @ 2.50GHz
    Memory: 4.02 GB / 11.87 GB
  Binaries:
    Node: 20.9.0 - C:\Program Files\nodejs\node.EXE
    npm: 10.1.0 - C:\Program Files\nodejs\npm.CMD
    pnpm: 7.29.1 - ~\AppData\Roaming\npm\pnpm.CMD
  Browsers:
    Edge: Chromium (127.0.2651.74)
    Internet Explorer: 11.0.19041.4355
  npmPackages:
    @sveltejs/kit: ^2.0.0 => 2.8.1
    bits-ui: ^1.0.0-next.64 => 1.0.0-next.64
    cmdk-sv: ^0.0.18 => 0.0.18
    formsnap: ^2.0.0-next.1 => 2.0.0-next.1
    lucide-svelte: ^0.460.1 => 0.460.1
    mode-watcher: ^0.4.1 => 0.4.1
    svelte: ^5.0.0 => 5.2.2
    svelte-radix: ^2.0.1 => 2.0.1
    svelte-sonner: ^0.3.28 => 0.3.28
    sveltekit-superforms: ^2.20.1 => 2.20.1
    vaul-svelte: ^1.0.0-next.3 => 1.0.0-next.3
```

### Severity

annoyance

**Comment by huntabyte** [comment]
If an issue only occurs when you add paraglide, perhaps the issue is with paraglide?
